### DISCUSSION TITLE
https url

### DISCUSSION THREAD

**Discussion by anonymous33rus** [thedevdojo/voyager#5387] [Q&A]
## Version information

 - Laravel: v8.31.0
 - Voyager: v1.4
 - PHP: 7.4
 - Database: . MySQL 5.7

## Description of problem
Mixed Content: The page at 'https://site.pro/admin/posts/create' was loaded over HTTPS, but requested an insecure stylesheet 'http://site.pro/admin/voyager-assets?path=js/skins/voyager/skin.min.css'. This request has been blocked; the content must be served over HTTPS.

## Additional context

How to solve this problem? the .env file specifies a domain with https. executed the command to the console (php artisan optimize
)  If you look at the html code of the page, you can see that https is not on all links. I ask for help.

<meta name="assets-path" content="http://site.pro/admin/voyager-assets"/>

<html>
<body>

<div class="panel widget center bgimage"
--
  | style="background-image:url(http://site.pro/admin/voyager-assets?path=images%2Fbg.jpg); background-size: cover; background-position: 0px;">

</body>
</html>

**Comment by MrCrayon** [comment]
It's something related to your server / Laravel configuration.

Try to configure Trusted proxies as explained in Laravel docs
https://laravel.com/docs/8.x/requests#configuring-trusted-proxies

**Reply by anonymous33rus** [reply]
Did not help

**Comment by howdu** [comment]
If you check the html source does the meta tag "assets-path" contain the http url?

Bu as @MrCrayon said this is standard Laravel:
`<meta name="assets-path" content="{{ route('voyager.voyager_assets') }}"/>`
